pkg/system/reconciler.go

```diff
@@ -194,6 +198,8 @@ func NewReconciler(
 	r.GCPBucketCreds.Name = r.Request.Name + "-gcp-bucket-creds"
 	r.GCPCloudCreds.Name = r.Request.Name + "-gcp-cloud-creds"
 	r.GCPCloudCreds.Spec.SecretRef.Name = r.Request.Name + "-gcp-cloud-creds-secret"
+	r.IsIBMCloud = false
```

Comment:
This can also be removed and replaced by util function call until the cloud credentials operator will support IBM COS.

Reply:
There is a util function, but it uses api calls to kube to determine that, so we run that in the reconcile loop.
I wonder if we should run that just once and then keep the result instead of all these wasteful calls every reconcile cycle...

Reply:
In any case I wouldn't call that util function in this ctor, right?

Reply:
I've meant to remove this value and call the util function that exists in the single place where we look at that value.
But if you are saying that we do not want to trigger another API call, then we can leave it like that.

Reply:
ok so I think the other comment I wrote here - https://github.com/noobaa/noobaa-operator/pull/446#pullrequestreview-536328625 is more relevant now.